cmd/speep/main.go

```diff
@@ -84,6 +85,7 @@ func HandleGet(getCmd *flag.FlagSet, all *bool, keyword *string) error {
 	if *keyword != "" {
 		shortcuts, err := shortcuts.GetShortcuts()
 		if err != nil {
+			err = fmt.Errorf("Failed to acquire the existing list: %w", err)
```

Comment:
Errors should not be capitalized unless it's a proper noun or an acronym

Reply:
Thank you all fixed!